src/planner/parabolic/ParabolicTimer.cpp

```diff
@@ -24,7 +24,7 @@ std::unique_ptr<aikido::trajectory::Spline> convertToSpline(
 {
   using aikido::statespace::GeodesicInterpolator;
 
-  if (nullptr == dynamic_cast<GeodesicInterpolator*>(
+  if (nullptr == dynamic_cast<const GeodesicInterpolator*>(
```

Comment:
I don't think an explicit comparison to `nullptr` is necessary. Instead, let's do something like
`if (!dynamic_pointer_cast<const GeodesicInterpolator>(interpolator))`?